### PR TITLE
4957 external sql

### PR DESCRIPTION
**Pull request by hooverlunch**
The spec failure is a very strange flapper. Runs fine individually.

### CODE REVIEW

File: app/assets/stylesheets/all/components/_settings.css.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+form.setting_form {
```
Comment (meltheadorable) [inline app/assets/stylesheets/all/components/_settings.css.scss]: YAY! Love seeing the CSS getting broken out more as we move forward.

File: app/models/results/sql_generator.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      select("answers.time_value AS time_value").
+      select("answers.latitude AS latitude").
+      select("answers.longitude AS longitude").
+      select("IFNULL(ao.canonical_name, co.canonical_name) AS choice_name").
+      select("option_sets.name AS option_set").
+      joins(Results::Join.list_to_sql(:users, :forms, :answers, :questionings,
+        :questions, :option_sets, :options, :choices)).
+      where("responses.mission_id = ?", mission.id).
+      order("responses.created_at DESC").
+      to_sql
```
Comment (meltheadorable) [inline app/models/results/sql_generator.rb]: Just wondering if there is any benefit to letting this query stay an activerecord relation and calling `to_sql` when we need SQL specifically. I guess if that need presents itself we could always refactor later, I was just trying to think through whether there's anywhere else where we're producing a complete result set like this where we would benefit from being able to re-use the relation.

File: spec/models/sms/sms_decoder_spec.rb [hunk 3]
```diff
@@ -6,25 +6,25 @@
 
   describe "authentication" do
     context "with SMS Authentication enabled" do
-      before(:all) { create_form(questions: %w(integer), authenticate_sms: true) }
+      let(:form) { create_form(questions: %w(integer), authenticate_sms: true) }
```
Comment (meltheadorable) [inline spec/models/sms/sms_decoder_spec.rb]: Thanks for making these changes, fingers crossed that we stop getting these spec errors on the SMS decoding! 🤞

File: app/models/results/sql_generator.rb (hunk 2 above)
Comment (hooverlunch) [inline app/models/results/sql_generator.rb]: Yeah probably not. Originally I think the mission scope was coming in this way, but not sure why, just trying to be cute I guess :o

Comment (meltheadorable) [inline app/models/results/sql_generator.rb]: That's what I thought, just checking 😄

**Pull request merged**